tests/cucumber/features/accounts.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Accounts
+
+  Background:
+    Given I am a new user
+
+  Scenario: Creating a new account
+    When I navigate to "/"
+    And I open the account modal
+    And I fill out the new account form
+    Then I am logged in
+    And I should see content "Sign Out"
+    And I should see content "Profile"
+
+  Scenario: Editing my profile
+    When I register an account
+    And I navigate to "/profile/edit"
+    Then I should not see a "Success" toast
+    When I fill out the profile edit form
+    Then I should see a "Success" toast
+
+  Scenario: Viewing a public profile
+    Given there is a profile in the database where "fullName" is "Test Title"
+    When I navigate to "/profiles/fakeid"
```

Comment:
I completely agree, would love a better solution here.  I think the real issue is that there's no way to navigate to other people's profiles through the interface yet, and cucumber is definitely only meant for testing the interface.